docker/dev/README.md

```diff
@@ -32,21 +34,21 @@ brew install make
 ## 2. Fork repository
 
 1. Go to [github.com/the-clothing-loop/website](https://github.com/the-clothing-loop/website) and name it something like "clothing-loop"
-2. Disable actions in fork
-3. Clone repo locally "git clone https://github.com/<your_username>/clothing-loop.git"
+2. Disable actions in fork (see https://github.com/orgs/community/discussions/26704)
+3. Clone repo locally "git clone https://github.com/<your_username>/clothing-loop.git" (in VS code)
```

Comment:
Using the built in vscode terminal is not a requirement I'd like to leave it to the individual developer which terminal application they use. 

If this was because you are running windows then an explanation of which terminal environment is required might make more sense (in 1.a.).